src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -137,3 +137,69 @@
         - name: --ids
           short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
 """
+
+helps['vm get-instance-view'] = """
+    type: command
+    short-summary: "Get's a vm including instance information (powerState)"
```

Comment:
Fixed.  And powerState is called out because it is likely confusing for newer customers.  Final text should be reviewed with service team.